src/dev/build/tasks/os_packages/docker_generator/run.ts

```diff
@@ -131,13 +131,6 @@ export async function runDockerGenerator(
     dockerBuildDir
   );
 
-  if (flags.ironbank) {
```

Comment:
This wasn't matching on anything and started throwing errors with the new copyAll implementation, which is probably a reasonable API change.

`resources` (static files) are used in parallel with `templates` but we ended up moving all the ironbank context files to templates.